src/payments/update_builder.rs

```diff
@@ -0,0 +1,121 @@
+use reqwest::Method;
+
+use crate::{
+    client::MercadoPagoClient,
+    common::{resolve_json, MercadoPagoRequestError},
+};
+
+use super::types::{PartialPaymentResult, PaymentResponse, PaymentStatus, PaymentUpdateOptions};
+
+/// Builder for update a payment
+///
+/// # Arguments
+///
+/// * `id` - Unique payment identifier, automatically generated by Mercado Pago.
+/// * `options` - Options to update payment.
+///
+/// # Example
+/// ```
+/// PaymentUpdateBuilder(
+///     8972364,
+///     PaymentUpdateOptions {
+///         status: Ok(PaymentStatus::Cancelled),
+///         ..Default::default()
+///     }
+/// )
+/// ```
+///
+/// # Docs
+/// <https://www.mercadopago.com.br/developers/pt/reference/payments/_payments_id/put>
+pub struct PaymentUpdateBuilder(pub u64, pub PaymentUpdateOptions);
```

Comment:
Acho que aqui podias considerar usar uma struct normal invés de tupla, structs de tupla são usados em situações que o propósito dos valores são possíveis de identificar pelo tipo do valor, mas aqui você já precisou documentar entao aq ta usando no lugar errado

```suggestion
pub struct PaymentUpdateBuilder {
    pub id: u64, 
    pub options: PaymentUpdateOptions
}
```

Reply:
Arrumado no commit [e34dfc0](https://github.com/vipago/mpago/pull/2/commits/e34dfc07e093bcc0380c23347dcb820f5a3babc5)